mongodb/map.jinja

```diff
@@ -1,14 +1,26 @@
 {% set mongodb = salt['grains.filter_by']({
     'default': {
-        'pkg': 'mongodb',
-        'service': 'mongodb',
-        'conf_file': '/etc/mongodb/mongodb.conf',
-        'config': {
-            'key': 'value',
-        },
+      'key': '7F0CEB10',
+      'keyserver': 'keyserver.ubuntu.com',
+      'pkgs': [
+        'mongodb-org',
+      ],
+      'pip_pkgs': [
+        'pymongo',
+      ],
+      'cluster_key_file': '/etc/mongodb_key',
+      'bind_ip': '127.0.0.1',
+      'port': '12017',
+      'enable_journal': True,
+      'install_pkgrepo': true,
+      'dbpath': '/var/lib/mongodb',
+      'logpath': '/var/log/mongodb/mongodb.log',
     },
     'Debian': {
+      'repo': 'deb http://downloads-distro.mongodb.org/repo/debian-sysvinit dist 10gen',
+      'repo_dist': 'dist',
```

Comment:
This was one of the weird things I struggled with in this module. Even though the dist is given in the deb line, the resulting line in the deb mirror conf would always end up being some default if "repo_dist" wasn't specified. I quickly tried again locally and it can't find the package when I remove the repo_dist line. I'm still unsure what the practical use of repo_dist is.